Google.Api.Generator/Generation/ServiceSettingsCodeGenerator.cs

```diff
@@ -151,7 +151,31 @@ IEnumerable<PropertyDeclarationSyntax> PerMethod(MethodDetails method)
                                 $"Retry delay multiplier: {retry.BackoffMultiplier}",
                                 $"Retry maximum delay: {(int)retry.MaxBackoff.TotalMilliseconds} milliseconds.",
                                 $"Maximum attempts: {(retry.MaxAttempts == int.MaxValue ? "Unlimited" : retry.MaxAttempts.ToString())}",
+                                GetRetriableErrorCodeDocs().ToArray(),
                                 timeoutRemark));
+
+                        IEnumerable<object> GetRetriableErrorCodeDocs()
+                        {
+                            var codes = method.MethodRetryStatusCodes.ToList();
+                            if (!codes.Any())
+                            {
+                                // This is a little unusual, in terms of "Here are the retry timing parameters, that will never be used..." but
+                                // I guess the settings could then be augmented with status codes. (And this does happen.)
+                                yield return "No status codes are retried.";
```

Comment:
Can't you get the retry status codes before adding the remark on the method above, and only add the remarks if there are some retriable codes?

Reply:
We could, but my point is that the remarks are still potentially useful - because I think users could fairly easily build new settings based on these ones, *only* changing the status codes, in which case they'd want to know the timing.

Reply:
Ah yes, I see, agreed.